### DISCUSSION TITLE
Alternative to setting logo/image on keybase?

### DISCUSSION THREAD

**Discussion by Cordtus** [ping-pub/explorer#157] [General]
Hey so keybase.io is broken (see issue linked below) and seems to have been sort of maybe abandoned... I broke my image on keybase so it's blank now, and no newcomers can add any image either. Can we have a method to manually add the image here like with Mintscan?

https://github.com/keybase/client/issues/25100

edit: I opened an issue on cosmos sdk that might be an alright idea. -- https://github.com/cosmos/cosmos-sdk/issues/12423

**Comment by liangping** [comment]
Yeah. That's a good idea.

**Reply by Cordtus** [reply]
edited OP with link to my issue thread if you have any ideas please comment there
